### DISCUSSION TITLE
g-code thumbnail error in Prusaslicer 2.5

### DISCUSSION THREAD

**Discussion by brew99** [mriscoc/Ender3V2S1#681] [Q&A] [answered]
So I'm trying to follow the wiki for generating a thumbnail in g-code via Prusaslicer, and am getting this error.
![Screenshot 2023-01-25 111845](https://user-images.githubusercontent.com/36134056/214694566-0b5beeea-7fff-4d68-a9ea-53ab3ea107eb.png)

Any idea what is going on?  I have verified python install and Pillow via below
![cmd](https://user-images.githubusercontent.com/36134056/214694686-9c575682-e11b-4e49-89f5-6dc75d9b7df8.png)

**Comment by mriscoc** [comment, marked as answer]
> Any idea what is going on? I have verified python install and Pillow via below

In your last picture there isn't Pillow installed 

See:
![image](https://user-images.githubusercontent.com/2745567/214762525-11009bd0-3aa8-4267-baa1-0caef06e0a03.png)

**Reply by brew99** [reply]
OMG, I must be blind. Thanks
